<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegWindow</class>
 <widget class="QWidget" name="RegWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="ledit_user">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledit_passwd">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ledit_age">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>571</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reg">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>注册</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn_man">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>男</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbtn_woman">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>女</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
